moved/src/types/state.rs

```diff
@@ -102,6 +103,85 @@ pub enum StateMessage {
     },
 }
 
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct BlockResponse {
+    /// the block number. null when its pending block.
```

Comment:
nit: same comment capitalization `/// The ...`

Reply:
Is there a type from the alloy library that can be used here instead of defining it ourselves?

Reply:
`alloy::consensus::Block`